runtime/crab/src/migration.rs

```diff
@@ -41,7 +41,18 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> frame_support::weights::Weight {
-	frame_support::weights::Weight::zero()
+	// substrate
+	use frame_support::PalletId;
+	use pallet_balances::WeightInfo;
+	use sp_runtime::traits::AccountIdConversion;
+
+	let staking: AccountId = PalletId(*b"da/staki").into_account_truncating();
+	let deprecate_staking: AccountId = PalletId(*b"dar/stak").into_account_truncating();
+
+	let _ = Balances::transfer_all(RuntimeOrigin::signed(deprecate_staking), staking, false);
```

Comment:
Have you ever checked the KTON asset under the deprecated account?

```rs
pub enum KtonStaking {}
impl darwinia_staking::Stake for KtonStaking {
	type AccountId = u32;
	type Item = Balance;

	fn stake(who: &Self::AccountId, item: Self::Item) -> sp_runtime::DispatchResult {
		Assets::transfer(
			RuntimeOrigin::signed(*who),
			0.into(),
			darwinia_staking::account_id(),  // here
			item,
		)
	}

	fn unstake(who: &Self::AccountId, item: Self::Item) -> sp_runtime::DispatchResult {
		Assets::transfer(
			RuntimeOrigin::signed(darwinia_staking::account_id()),
			0.into(),
			*who,
			item,
		)
	}
}
```

Reply:
Done.